src/forms/__snapshots__/textInput.test.jsx.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TextInput snapshot check matches snapshot 1`] = `
+<div>
+  <label
+    className="label--field required"
+    htmlFor="superhero"
+  >
+    Super Hero
+  </label>
+  <input
+    className="field--error"
+    id="superhero"
+    maxLength={20}
+    name="superhero"
+    onChange={[Function]}
+    required={true}
```

Comment:
is this the way it should be rendered `required={true}` rather than just a required `attr`?
same for maxLength...